retriever/lib/datasets.py

```diff
@@ -27,19 +34,27 @@ def datasets(keywords=None, licenses=None):
                 script_keywords = script_keywords.lower()
                 for k in keywords:
                     if script_keywords.find(k.lower()) != -1:
-                        result_scripts.add(script)
+                        result_scripts_offline.add(script)
                         break
-    return sorted(list(result_scripts), key=lambda s: s.name.lower())
+    # The offline scripts filtered by params
+    result_scripts_offline = sorted(list(result_scripts_offline), key=lambda s: s.name.lower())
+    # The scripts present in upstream retriever repository filtered by params
+    result_retriever_scripts_online = get_dataset_names_upstream(keywords, licenses, repo=RETRIEVER_REPOSITORY)
```

Comment:
The variable names are still lone. Use smaller names for these group of variables. 
The comments should help in telling us what the variable name holds